hazelcast/src/test/java/com/hazelcast/map/LocalIndexStatsTest.java

```diff
@@ -490,7 +490,7 @@ public void testInsertsTracking() {
             totalMeasuredLatency += Timer.nanosElapsed(startNanos);
 
             assertEquals(i, keyStats().getInsertCount());
-            assertTrue(keyStats().getTotalInsertLatency() > previousTotalInsertLatency);
+            assertTrue(keyStats().getTotalInsertLatency() >= previousTotalInsertLatency);
```

Comment:
`assertTrue` doesn't show `expected` and `actual` values thus making debugging harder, I would suggest to migrate  it to assert: 
```java
assertThat(keyStats().getTotalInsertLatency()).isGreaterThan(previousTotalInsertLatency);
assertThat(keyStats().getTotalInsertLatency()).isLessThanOrEqualTo(totalMeasuredLatency);
```

This way in case of a failure we see something like:
```
Expecting:
 <26833L>
to be greater than:
 <743833L> 
```

Reply:
Thanks! I have changed like that